src/curobo_rviz.cpp

```diff
@@ -17,14 +20,17 @@ namespace curobo_rviz
     auto options = rclcpp::NodeOptions().arguments(
         {"--ros-args", "--remap", "__node:=rviz_push_button_node", "--"});
     node_ = std::make_shared<rclcpp::Node>("_", options);
-
-    // Prepare msg
-    msg_.data = true;
+    param_client_ = std::make_shared<rclcpp::SyncParametersClient>(node_, "curobo_gen_traj");
+    while (!param_client_->wait_for_service(std::chrono::seconds(3))) {
+      RCLCPP_INFO(node_->get_logger(), "service not available, waiting again...");
+    }
 
     // Connect SpinBox and DoubleSpinBox to slots
     connect(ui_->spinBox_1, SIGNAL(valueChanged(int)), this, SLOT(updateMaxAttempts(int)));
     connect(ui_->doubleSpinBox_1, SIGNAL(valueChanged(double)), this, SLOT(updateTimeout(double)));
     connect(ui_->doubleSpinBox_2, SIGNAL(valueChanged(double)), this, SLOT(updateTimeDilationFactor(double)));
+    connect(ui_->doubleSpinBox_3, SIGNAL(valueChanged(double)), this, SLOT(updateVoxelSize(double)));
+    connect(ui_->doubleSpinBox_4, SIGNAL(valueChanged(double)), this, SLOT(updateCollisionActivationDistance(double)));
```

Comment:
Could the UI elements have more distinguishable names instead?

Reply:
+1